buildscript/build.go

```diff
@@ -25,6 +27,45 @@ var toolNames = []string{
 	"mongofiles",
 }
 
+// SAModTidy runs go mod tidy and ensure no changes were made.
+// Copied from mongohouse: https://github.com/10gen/mongohouse/blob/333308814f96a0909c8125f71af7748b263e3263/buildscript/sa.go#L72
+func SAModTidy(ctx *task.Context) error {
```

Comment:
since we're aiming to eventually move more of our static analysis stuff into go code, perhaps we can put this into a separate `sa.go` file instead of just bundling it here?